rootfs/etc/nginx/lua/certificate.lua

```diff
@@ -217,6 +219,18 @@ function _M.configured_for_current_request()
   return ngx.ctx.cert_configured_for_current_request
 end
 
+function _M.set_cache_size(size)
+  local cache, err = lrucache.new(size)
+  if err then
+    ngx.log(ngx.ERR, string.format("failed to create the certificate cache: %s", tostring(err)))
```

Comment:
I think we can use ` if not cache then` , because if  lrucache instance create failed, it will be `nil`

And for L225, we don't need `tostring`,  `err` is already of type string, right?

Reply:
Sure, updated.

Reply:
@tao12345666333 Could you take a second look?